llvm/unittests/Transforms/Vectorize/VPlanTest.cpp

```diff
@@ -706,13 +706,15 @@ TEST_F(VPBasicBlockTest, reassociateBlocks) {
 
 #if !defined(NDEBUG) || defined(LLVM_ENABLE_DUMP)
 TEST_F(VPBasicBlockTest, print) {
-  VPInstruction *TC = new VPInstruction(Instruction::Add, {});
+  VPInstruction *TC = new VPInstruction(Instruction::PHI, {});
```

Comment:
To escape num operands check?
Considered scalar, w/o operands.

Reply:
Yes. The tests here just check the various print implementations, the actual opcodes/operands don't matter much, other than matching the print checks.

There's a chicken-egg problem here, as we need to get a VPValue before calling getPlan.